at_secondary/at_secondary_server/lib/src/verb/handler/config_verb_handler.dart

```diff
@@ -83,6 +83,43 @@ class ConfigVerbHandler extends AbstractVerbHandler {
           result = 'unknown operation';
           break;
       }
+
+      switch (setOperation) {
+        case 'set':
+          if (AtSecondaryConfig.testingMode &&
```

Comment:
@srieteja we have to check with Gary if he is fine with making "testingMode" part of the config

Reply:
Yes I'm good with this. It should default to 'false' of course, and we will need to inject it into the docker env for the secondaries we use in testing